In [47]:
import matplotlib.pyplot as plt
import math
import pandas as pd
import numpy as np
import plotly.express as px 
import plotly.graph_objects as go
import plotly
from decimal import * #gives us perfect precision decimal points

# Reverse Process
Rather than reverse and add, this notebook demonstrates reverse and subtract.

In [77]:
pal = 196
n = 0
n_final = 1000

# generator, break condition removed as we can save computation time
def lychrel_generator(start) :
    a = start
    b = abs(a - int(str(a)[::-1]))
    while True :
        yield a
        a , b = b , abs(b - int(str(b)[::-1]))

lychrel_gen = lychrel_generator(pal)


title_text = "Frequency Of Digits for " + str(n_final) + " Steps of the Lychrel Process on 196" 
image_title = "images/196DigitDistributionForN" + str(n_final)

In [113]:
data = pd.read_csv('datasets/LychrelTo100000Integers.txt',  header=None)
data = data.values.tolist()[0][1:]

In [109]:

valid = True

worked = 0
didnt_work = 0

for lychrel in data:
    lychrel_gen = lychrel_generator(lychrel)
    i=0
    for num in lychrel_gen:
        if (i > n_final or num == int(str(num)[::-1])):
            found_text = "Palindrome found for " + str(lychrel) + " at  " + str(num) + " after " + str(i) + " steps."
            print(found_text if num == int(str(num)[::-1]) else "limit exceeded for " + str(lychrel))
            valid = num == int(str(num)[::-1])
            if valid:
                worked+=1
            else:
                didnt_work+=1
            break
        else:
            #print(i, num)
            i+=1
print(valid)
print("The proportion of success is", str(worked/(worked+didnt_work) ))

Palindrome found for 196 at  99 after 2 steps.
Palindrome found for 295 at  99 after 3 steps.
Palindrome found for 394 at  99 after 1 steps.
Palindrome found for 493 at  99 after 1 steps.
Palindrome found for 592 at  99 after 3 steps.
Palindrome found for 689 at  99 after 3 steps.
Palindrome found for 691 at  99 after 2 steps.
Palindrome found for 788 at  99 after 1 steps.
Palindrome found for 790 at  99 after 4 steps.
Palindrome found for 879 at  99 after 1 steps.
Palindrome found for 887 at  99 after 1 steps.
Palindrome found for 978 at  99 after 1 steps.
Palindrome found for 986 at  99 after 3 steps.
Palindrome found for 1495 at  999 after 6 steps.
Palindrome found for 1497 at  999 after 6 steps.
Palindrome found for 1585 at  999 after 11 steps.
Palindrome found for 1587 at  999 after 8 steps.
Palindrome found for 1675 at  909 after 4 steps.
Palindrome found for 1677 at  999 after 7 steps.
Palindrome found for 1765 at  99 after 7 steps.
Palindrome found for 1767 at  909 after 5 step

limit exceeded for 12965
Palindrome found for 13087 at  9999 after 6 steps.
limit exceeded for 13097
limit exceeded for 13197
Palindrome found for 13287 at  9999 after 6 steps.
Palindrome found for 13383 at  9999 after 3 steps.
limit exceeded for 13392
limit exceeded for 13393
limit exceeded for 13397
Palindrome found for 13508 at  9999 after 6 steps.
limit exceeded for 13523
Palindrome found for 13533 at  9999 after 5 steps.
Palindrome found for 13547 at  9999 after 7 steps.
limit exceeded for 13553
limit exceeded for 13555
limit exceeded for 13596
limit exceeded for 13597
Palindrome found for 13608 at  9999 after 6 steps.
limit exceeded for 13623
Palindrome found for 13633 at  9999 after 5 steps.
Palindrome found for 13638 at  9999 after 4 steps.
Palindrome found for 13667 at  9999 after 8 steps.
limit exceeded for 13694
limit exceeded for 13705
limit exceeded for 13716
Palindrome found for 13718 at  9009 after 6 steps.
limit exceeded for 13753
limit exceeded for 13755
Palindrome fou

Palindrome found for 17787 at  9999 after 7 steps.
limit exceeded for 17807
Palindrome found for 17813 at  9999 after 8 steps.
limit exceeded for 17838
Palindrome found for 17848 at  9999 after 6 steps.
Palindrome found for 17867 at  9009 after 5 steps.
Palindrome found for 17897 at  9999 after 9 steps.
limit exceeded for 17907
Palindrome found for 17913 at  9999 after 8 steps.
Palindrome found for 17915 at  9999 after 11 steps.
limit exceeded for 17957
limit exceeded for 17965
limit exceeded for 17989
limit exceeded for 18037
limit exceeded for 18047
limit exceeded for 18056
Palindrome found for 18057 at  9999 after 5 steps.
limit exceeded for 18096
Palindrome found for 18097 at  9999 after 7 steps.
limit exceeded for 18147
Palindrome found for 18157 at  9999 after 5 steps.
Palindrome found for 18180 at  9999 after 1 steps.
Palindrome found for 18194 at  9999 after 10 steps.
limit exceeded for 18237
Palindrome found for 18257 at  9999 after 5 steps.
limit exceeded for 18267
Palindrome

limit exceeded for 23607
Palindrome found for 23622 at  99 after 6 steps.
Palindrome found for 23637 at  9999 after 2 steps.
Palindrome found for 23666 at  9999 after 11 steps.
limit exceeded for 23693
Palindrome found for 23704 at  9999 after 12 steps.
limit exceeded for 23715
limit exceeded for 23717
limit exceeded for 23752
limit exceeded for 23754
limit exceeded for 23756
Palindrome found for 23757 at  5445 after 3 steps.
limit exceeded for 23782
Palindrome found for 23786 at  9999 after 6 steps.
limit exceeded for 23812
Palindrome found for 23846 at  9009 after 4 steps.
limit exceeded for 23852
limit exceeded for 23877
Palindrome found for 23887 at  54945 after 1 steps.
limit exceeded for 23902
Palindrome found for 23934 at  9999 after 5 steps.
Palindrome found for 23938 at  9999 after 2 steps.
Palindrome found for 23946 at  9009 after 4 steps.
limit exceeded for 23952
limit exceeded for 23954
Palindrome found for 23996 at  9009 after 6 steps.
Palindrome found for 24076 at  9999 a

limit exceeded for 30746
Palindrome found for 30781 at  9999 after 6 steps.
limit exceeded for 30783
Palindrome found for 30785 at  9009 after 3 steps.
Palindrome found for 30786 at  9999 after 12 steps.
Palindrome found for 30841 at  9999 after 7 steps.
Palindrome found for 30849 at  9009 after 6 steps.
limit exceeded for 30875
Palindrome found for 30881 at  9999 after 6 steps.
limit exceeded for 30889
Palindrome found for 30929 at  9999 after 9 steps.
Palindrome found for 30931 at  9999 after 12 steps.
Palindrome found for 30959 at  9999 after 6 steps.
Palindrome found for 30961 at  9999 after 8 steps.
Palindrome found for 30963 at  5445 after 2 steps.
Palindrome found for 30967 at  9009 after 6 steps.
limit exceeded for 30975
Palindrome found for 30979 at  9999 after 11 steps.
Palindrome found for 30981 at  9999 after 6 steps.
limit exceeded for 30983
limit exceeded for 31079
limit exceeded for 31348
limit exceeded for 31379
Palindrome found for 31389 at  9999 after 11 steps.
Palind

limit exceeded for 35933
Palindrome found for 35969 at  9999 after 7 steps.
limit exceeded for 35975
limit exceeded for 35983
Palindrome found for 35999 at  9009 after 6 steps.
limit exceeded for 36029
limit exceeded for 36055
Palindrome found for 36065 at  9999 after 5 steps.
limit exceeded for 36074
limit exceeded for 36075
Palindrome found for 36165 at  9999 after 5 steps.
limit exceeded for 36175
limit exceeded for 36255
limit exceeded for 36275
limit exceeded for 36285
limit exceeded for 36329
Palindrome found for 36339 at  9999 after 5 steps.
limit exceeded for 36351
Palindrome found for 36360 at  9999 after 3 steps.
Palindrome found for 36361 at  9999 after 5 steps.
Palindrome found for 36365 at  9999 after 5 steps.
limit exceeded for 36375
Palindrome found for 36381 at  9009 after 5 steps.
Palindrome found for 36388 at  5445 after 3 steps.
limit exceeded for 36419
limit exceeded for 36429
Palindrome found for 36469 at  9999 after 2 steps.
Palindrome found for 36488 at  5445 aft

limit exceeded for 40574
Palindrome found for 40580 at  9999 after 6 steps.
Palindrome found for 40582 at  9999 after 6 steps.
limit exceeded for 40635
limit exceeded for 40650
Palindrome found for 40660 at  9999 after 11 steps.
limit exceeded for 40665
limit exceeded for 40694
limit exceeded for 40712
Palindrome found for 40725 at  9999 after 11 steps.
Palindrome found for 40732 at  9999 after 12 steps.
limit exceeded for 40743
Palindrome found for 40745 at  9009 after 4 steps.
Palindrome found for 40780 at  9999 after 6 steps.
Palindrome found for 40782 at  9999 after 6 steps.
limit exceeded for 40784
limit exceeded for 40785
Palindrome found for 40840 at  9009 after 4 steps.
limit exceeded for 40848
limit exceeded for 40874
Palindrome found for 40880 at  9999 after 6 steps.
Palindrome found for 40888 at  9999 after 9 steps.
limit exceeded for 40928
limit exceeded for 40930
Palindrome found for 40958 at  9999 after 6 steps.
Palindrome found for 40960 at  9999 after 11 steps.
Palindro

limit exceeded for 45974
Palindrome found for 45982 at  9999 after 12 steps.
limit exceeded for 45998
limit exceeded for 46009
Palindrome found for 46028 at  9009 after 4 steps.
Palindrome found for 46054 at  99 after 6 steps.
Palindrome found for 46073 at  9009 after 1 steps.
Palindrome found for 46074 at  99 after 6 steps.
limit exceeded for 46099
Palindrome found for 46174 at  99 after 6 steps.
Palindrome found for 46254 at  99 after 6 steps.
Palindrome found for 46274 at  99 after 6 steps.
limit exceeded for 46284
Palindrome found for 46328 at  9009 after 4 steps.
limit exceeded for 46338
Palindrome found for 46350 at  9009 after 4 steps.
Palindrome found for 46360 at  9999 after 4 steps.
Palindrome found for 46374 at  99 after 6 steps.
limit exceeded for 46380
Palindrome found for 46387 at  9009 after 7 steps.
limit exceeded for 46418
Palindrome found for 46428 at  9009 after 4 steps.
limit exceeded for 46459
Palindrome found for 46468 at  9999 after 4 steps.
limit exceeded for 46

limit exceeded for 50742
limit exceeded for 50744
Palindrome found for 50781 at  9999 after 6 steps.
Palindrome found for 50783 at  9999 after 6 steps.
limit exceeded for 50784
Palindrome found for 50847 at  9999 after 10 steps.
Palindrome found for 50873 at  9009 after 6 steps.
Palindrome found for 50887 at  9009 after 3 steps.
limit exceeded for 50927
Palindrome found for 50957 at  9999 after 3 steps.
Palindrome found for 50961 at  9999 after 11 steps.
Palindrome found for 50965 at  5445 after 2 steps.
Palindrome found for 50973 at  9009 after 6 steps.
limit exceeded for 50977
Palindrome found for 50978 at  9009 after 5 steps.
Palindrome found for 50981 at  9999 after 6 steps.
limit exceeded for 51058
limit exceeded for 51077
limit exceeded for 51346
limit exceeded for 51377
limit exceeded for 51387
Palindrome found for 51418 at  9999 after 3 steps.
limit exceeded for 51427
Palindrome found for 51438 at  9009 after 7 steps.
limit exceeded for 51446
Palindrome found for 51467 at  9999

Palindrome found for 60960 at  9009 after 2 steps.
Palindrome found for 60964 at  9999 after 8 steps.
limit exceeded for 60972
limit exceeded for 60976
limit exceeded for 60977
limit exceeded for 60980
Palindrome found for 61057 at  9009 after 4 steps.
Palindrome found for 61076 at  5445 after 2 steps.
limit exceeded for 61149
limit exceeded for 61159
limit exceeded for 61279
limit exceeded for 61345
Palindrome found for 61376 at  5445 after 2 steps.
limit exceeded for 61386
Palindrome found for 61417 at  9999 after 1 steps.
Palindrome found for 61426 at  99 after 6 steps.
Palindrome found for 61437 at  9999 after 11 steps.
limit exceeded for 61445
limit exceeded for 61466
Palindrome found for 61476 at  5445 after 2 steps.
limit exceeded for 61479
limit exceeded for 61488
limit exceeded for 61523
Palindrome found for 61558 at  9999 after 10 steps.
limit exceeded for 61562
Palindrome found for 61568 at  9999 after 3 steps.
Palindrome found for 61570 at  9009 after 2 steps.
limit exceede

limit exceeded for 65382
limit exceeded for 65395
Palindrome found for 65405 at  9999 after 4 steps.
limit exceeded for 65426
limit exceeded for 65436
limit exceeded for 65439
limit exceeded for 65448
limit exceeded for 65467
limit exceeded for 65476
limit exceeded for 65487
limit exceeded for 65495
Palindrome found for 65518 at  9999 after 7 steps.
Palindrome found for 65522 at  9999 after 11 steps.
Palindrome found for 65528 at  9999 after 12 steps.
Palindrome found for 65530 at  9999 after 9 steps.
limit exceeded for 65571
limit exceeded for 65572
Palindrome found for 65592 at  9009 after 4 steps.
limit exceeded for 65609
limit exceeded for 65613
limit exceeded for 65639
Palindrome found for 65642 at  9009 after 4 steps.
Palindrome found for 65683 at  9009 after 3 steps.
limit exceeded for 65699
Palindrome found for 65730 at  9999 after 9 steps.
limit exceeded for 65732
Palindrome found for 65733 at  9009 after 7 steps.
limit exceeded for 65762
Palindrome found for 65822 at  9999 af

Palindrome found for 69722 at  9999 after 8 steps.
Palindrome found for 69762 at  9999 after 11 steps.
Palindrome found for 69782 at  9009 after 4 steps.
Palindrome found for 69783 at  9999 after 10 steps.
Palindrome found for 69792 at  9999 after 4 steps.
Palindrome found for 69813 at  9999 after 12 steps.
Palindrome found for 69823 at  9009 after 5 steps.
Palindrome found for 69842 at  9999 after 6 steps.
limit exceeded for 69846
limit exceeded for 69872
Palindrome found for 69882 at  9009 after 4 steps.
Palindrome found for 69886 at  99 after 6 steps.
Palindrome found for 69895 at  9999 after 1 steps.
limit exceeded for 69926
Palindrome found for 69932 at  9009 after 6 steps.
Palindrome found for 69940 at  9999 after 6 steps.
limit exceeded for 69956
Palindrome found for 69964 at  9009 after 7 steps.
limit exceeded for 69976
limit exceeded for 69977
Palindrome found for 69993 at  9999 after 3 steps.
Palindrome found for 70066 at  5445 after 2 steps.
limit exceeded for 70069
Palindro

Palindrome found for 75905 at  9999 after 3 steps.
limit exceeded for 75913
Palindrome found for 75921 at  9999 after 8 steps.
Palindrome found for 75925 at  9009 after 7 steps.
limit exceeded for 75926
limit exceeded for 75965
limit exceeded for 75971
Palindrome found for 75989 at  9009 after 7 steps.
Palindrome found for 75995 at  9999 after 7 steps.
limit exceeded for 76006
Palindrome found for 76009 at  9999 after 8 steps.
Palindrome found for 76025 at  9999 after 10 steps.
Palindrome found for 76051 at  9999 after 7 steps.
Palindrome found for 76061 at  9999 after 2 steps.
Palindrome found for 76070 at  9999 after 8 steps.
Palindrome found for 76071 at  9009 after 5 steps.
limit exceeded for 76096
Palindrome found for 76099 at  9009 after 7 steps.
Palindrome found for 76108 at  5445 after 2 steps.
Palindrome found for 76161 at  9999 after 2 steps.
Palindrome found for 76171 at  9009 after 5 steps.
Palindrome found for 76188 at  9999 after 11 steps.
limit exceeded for 76198
limit e

limit exceeded for 80279
limit exceeded for 80287
limit exceeded for 80289
Palindrome found for 80299 at  9009 after 4 steps.
Palindrome found for 80349 at  9009 after 4 steps.
Palindrome found for 80353 at  9009 after 2 steps.
limit exceeded for 80358
limit exceeded for 80369
limit exceeded for 80378
Palindrome found for 80384 at  9999 after 6 steps.
Palindrome found for 80394 at  9009 after 6 steps.
limit exceeded for 80425
limit exceeded for 80434
limit exceeded for 80439
limit exceeded for 80445
Palindrome found for 80453 at  9009 after 2 steps.
limit exceeded for 80474
Palindrome found for 80484 at  9999 after 6 steps.
limit exceeded for 80487
limit exceeded for 80488
limit exceeded for 80489
limit exceeded for 80496
Palindrome found for 80529 at  9999 after 11 steps.
Palindrome found for 80531 at  9999 after 6 steps.
limit exceeded for 80539
limit exceeded for 80558
Palindrome found for 80566 at  9999 after 8 steps.
limit exceeded for 80569
limit exceeded for 80570
Palindrome fou

limit exceeded for 83945
limit exceeded for 83984
limit exceeded for 83990
limit exceeded for 84009
limit exceeded for 84019
Palindrome found for 84025 at  9009 after 7 steps.
limit exceeded for 84028
limit exceeded for 84029
Palindrome found for 84044 at  9999 after 4 steps.
Palindrome found for 84070 at  54945 after 2 steps.
limit exceeded for 84080
limit exceeded for 84090
Palindrome found for 84099 at  9999 after 4 steps.
limit exceeded for 84109
limit exceeded for 84117
limit exceeded for 84119
Palindrome found for 84127 at  9999 after 11 steps.
limit exceeded for 84129
limit exceeded for 84159
limit exceeded for 84179
limit exceeded for 84180
limit exceeded for 84190
Palindrome found for 84199 at  9999 after 4 steps.
limit exceeded for 84229
Palindrome found for 84239 at  9009 after 1 steps.
Palindrome found for 84247 at  9999 after 1 steps.
Palindrome found for 84249 at  9999 after 1 steps.
limit exceeded for 84259
Palindrome found for 84270 at  54945 after 2 steps.
Palindrome f

limit exceeded for 89109
limit exceeded for 89129
Palindrome found for 89130 at  9999 after 10 steps.
Palindrome found for 89140 at  9999 after 3 steps.
limit exceeded for 89149
Palindrome found for 89157 at  9009 after 4 steps.
limit exceeded for 89159
limit exceeded for 89167
limit exceeded for 89169
limit exceeded for 89209
Palindrome found for 89220 at  9009 after 7 steps.
Palindrome found for 89239 at  5445 after 2 steps.
Palindrome found for 89240 at  9999 after 3 steps.
limit exceeded for 89250
limit exceeded for 89269
Palindrome found for 89270 at  9009 after 3 steps.
limit exceeded for 89279
limit exceeded for 89287
Palindrome found for 89289 at  9009 after 1 steps.
Palindrome found for 89296 at  9999 after 5 steps.
Palindrome found for 89297 at  9999 after 1 steps.
Palindrome found for 89304 at  9999 after 7 steps.
Palindrome found for 89330 at  9999 after 10 steps.
Palindrome found for 89340 at  9999 after 3 steps.
limit exceeded for 89349
limit exceeded for 89353
limit exce

limit exceeded for 92469
limit exceeded for 92475
limit exceeded for 92479
limit exceeded for 92494
Palindrome found for 92508 at  9999 after 11 steps.
Palindrome found for 92510 at  9009 after 4 steps.
limit exceeded for 92518
limit exceeded for 92537
limit exceeded for 92545
limit exceeded for 92548
limit exceeded for 92555
limit exceeded for 92559
limit exceeded for 92568
limit exceeded for 92569
limit exceeded for 92598
Palindrome found for 92610 at  9009 after 4 steps.
limit exceeded for 92618
Palindrome found for 92628 at  9999 after 1 steps.
limit exceeded for 92636
limit exceeded for 92640
Palindrome found for 92647 at  9009 after 5 steps.
limit exceeded for 92648
limit exceeded for 92666
limit exceeded for 92668
limit exceeded for 92700
Palindrome found for 92720 at  9999 after 5 steps.
Palindrome found for 92728 at  9999 after 1 steps.
limit exceeded for 92748
limit exceeded for 92758
Palindrome found for 92760 at  9009 after 6 steps.
Palindrome found for 92789 at  5445 after

limit exceeded for 96108
limit exceeded for 96138
limit exceeded for 96158
Palindrome found for 96178 at  9009 after 1 steps.
limit exceeded for 96186
limit exceeded for 96188
Palindrome found for 96196 at  9009 after 3 steps.
limit exceeded for 96198
limit exceeded for 96208
Palindrome found for 96218 at  9999 after 4 steps.
limit exceeded for 96219
limit exceeded for 96226
Palindrome found for 96228 at  9009 after 4 steps.
limit exceeded for 96238
Palindrome found for 96268 at  9999 after 1 steps.
Palindrome found for 96279 at  99 after 6 steps.
limit exceeded for 96298
limit exceeded for 96308
Palindrome found for 96317 at  9999 after 3 steps.
limit exceeded for 96319
Palindrome found for 96323 at  9009 after 6 steps.
Palindrome found for 96333 at  9999 after 8 steps.
limit exceeded for 96349
Palindrome found for 96378 at  9009 after 1 steps.
Palindrome found for 96382 at  9009 after 6 steps.
Palindrome found for 96387 at  9009 after 5 steps.
limit exceeded for 96398
Palindrome foun

limit exceeded for 98658
Palindrome found for 98666 at  9009 after 7 steps.
limit exceeded for 98677
limit exceeded for 98678
limit exceeded for 98696
Palindrome found for 98698 at  9009 after 1 steps.
Palindrome found for 98700 at  9999 after 11 steps.
Palindrome found for 98729 at  5445 after 2 steps.
limit exceeded for 98758
limit exceeded for 98778
Palindrome found for 98788 at  9999 after 1 steps.
Palindrome found for 98790 at  9999 after 11 steps.
limit exceeded for 98803
Palindrome found for 98820 at  9009 after 4 steps.
Palindrome found for 98830 at  9999 after 4 steps.
Palindrome found for 98853 at  9999 after 8 steps.
limit exceeded for 98858
limit exceeded for 98877
Palindrome found for 98893 at  9009 after 5 steps.
Palindrome found for 98933 at  9999 after 6 steps.
Palindrome found for 98957 at  9009 after 7 steps.
Palindrome found for 98963 at  9999 after 9 steps.
Palindrome found for 98968 at  9999 after 11 steps.
limit exceeded for 98971
Palindrome found for 98983 at  99

For the sample size of 10000 integers the success rate of this method is 0.947, yet by increasing it to 100000 the success rate drops to 0.0.540531561461794. It is interesting to note that of those that do work, almost all of them seem to converge to a number of all 9s (99, 999, 9999, etc) or 0s bookended by 9s (909, 9009, 90009, etc). There also seem to be a handful of instances of numbers converging on 5445 or 54945.

It is also interesting to note that this method works for every single 3 digit lychrel.

Interestingly, it would appear that many of those that dont work fall into the same thread of continually alternating between 2178 and 6534, as by the reverse process each of these numbers subtracts into the other (8712 - 2178 = 6534, while 6534 - 4356 = 2178). Another thread I found that only occurs towards the upper end of the set (still interspersed with the previous example) is the similar alternation between 65934 and 21978.

In [115]:
n_final = 10000

#store lychrel numbers
revlychrel_list = []

#store results
step_list = []
result_list = []

for n in range(n_final):
    lychrel_gen = lychrel_generator(n)
    i=0
    for num in lychrel_gen:
        if (i > n_final or num == int(str(num)[::-1])):
            found_text = "Palindrome found for " + str(n) + " at  " + str(num) + " after " + str(i) + " steps."
            print(found_text if num == int(str(num)[::-1]) else "limit exceeded for " + str(n))
            valid = num == int(str(num)[::-1])
            if valid:
                step_list.append(i)
                result_list.append(num)
            else:
                revlychrel_list.append(n)
            break
        else:
            #print(i, num)
            i+=1

Palindrome found for 0 at  0 after 0 steps.
Palindrome found for 1 at  1 after 0 steps.
Palindrome found for 2 at  2 after 0 steps.
Palindrome found for 3 at  3 after 0 steps.
Palindrome found for 4 at  4 after 0 steps.
Palindrome found for 5 at  5 after 0 steps.
Palindrome found for 6 at  6 after 0 steps.
Palindrome found for 7 at  7 after 0 steps.
Palindrome found for 8 at  8 after 0 steps.
Palindrome found for 9 at  9 after 0 steps.
Palindrome found for 10 at  9 after 1 steps.
Palindrome found for 11 at  11 after 0 steps.
Palindrome found for 12 at  9 after 1 steps.
Palindrome found for 13 at  9 after 5 steps.
Palindrome found for 14 at  9 after 3 steps.
Palindrome found for 15 at  9 after 4 steps.
Palindrome found for 16 at  9 after 2 steps.
Palindrome found for 17 at  9 after 2 steps.
Palindrome found for 18 at  9 after 4 steps.
Palindrome found for 19 at  9 after 3 steps.
Palindrome found for 20 at  9 after 5 steps.
Palindrome found for 21 at  9 after 1 steps.
Palindrome found fo

limit exceeded for 1023
Palindrome found for 1024 at  909 after 7 steps.
Palindrome found for 1025 at  99 after 10 steps.
Palindrome found for 1026 at  99 after 4 steps.
Palindrome found for 1027 at  999 after 9 steps.
Palindrome found for 1028 at  999 after 12 steps.
Palindrome found for 1029 at  909 after 3 steps.
Palindrome found for 1030 at  99 after 6 steps.
Palindrome found for 1031 at  99 after 6 steps.
Palindrome found for 1032 at  99 after 10 steps.
Palindrome found for 1033 at  909 after 7 steps.
limit exceeded for 1034
Palindrome found for 1035 at  999 after 11 steps.
Palindrome found for 1036 at  99 after 5 steps.
Palindrome found for 1037 at  999 after 8 steps.
Palindrome found for 1038 at  909 after 5 steps.
Palindrome found for 1039 at  99 after 12 steps.
Palindrome found for 1040 at  99 after 4 steps.
Palindrome found for 1041 at  99 after 4 steps.
Palindrome found for 1042 at  909 after 4 steps.
Palindrome found for 1043 at  999 after 10 steps.
Palindrome found for 104

limit exceeded for 1243
Palindrome found for 1244 at  909 after 7 steps.
Palindrome found for 1245 at  99 after 10 steps.
Palindrome found for 1246 at  99 after 4 steps.
Palindrome found for 1247 at  999 after 9 steps.
Palindrome found for 1248 at  999 after 12 steps.
Palindrome found for 1249 at  909 after 3 steps.
Palindrome found for 1250 at  99 after 6 steps.
Palindrome found for 1251 at  99 after 6 steps.
Palindrome found for 1252 at  99 after 10 steps.
Palindrome found for 1253 at  909 after 7 steps.
limit exceeded for 1254
Palindrome found for 1255 at  999 after 11 steps.
Palindrome found for 1256 at  99 after 5 steps.
Palindrome found for 1257 at  999 after 8 steps.
Palindrome found for 1258 at  909 after 5 steps.
Palindrome found for 1259 at  99 after 12 steps.
Palindrome found for 1260 at  99 after 4 steps.
Palindrome found for 1261 at  99 after 4 steps.
Palindrome found for 1262 at  909 after 4 steps.
Palindrome found for 1263 at  999 after 10 steps.
Palindrome found for 126

limit exceeded for 1463
Palindrome found for 1464 at  909 after 7 steps.
Palindrome found for 1465 at  99 after 10 steps.
Palindrome found for 1466 at  99 after 4 steps.
Palindrome found for 1467 at  999 after 9 steps.
Palindrome found for 1468 at  999 after 12 steps.
Palindrome found for 1469 at  909 after 3 steps.
Palindrome found for 1470 at  99 after 6 steps.
Palindrome found for 1471 at  99 after 6 steps.
Palindrome found for 1472 at  99 after 10 steps.
Palindrome found for 1473 at  909 after 7 steps.
limit exceeded for 1474
Palindrome found for 1475 at  999 after 11 steps.
Palindrome found for 1476 at  99 after 5 steps.
Palindrome found for 1477 at  999 after 8 steps.
Palindrome found for 1478 at  909 after 5 steps.
Palindrome found for 1479 at  99 after 12 steps.
Palindrome found for 1480 at  99 after 4 steps.
Palindrome found for 1481 at  99 after 4 steps.
Palindrome found for 1482 at  909 after 4 steps.
Palindrome found for 1483 at  999 after 10 steps.
Palindrome found for 148

limit exceeded for 1672
Palindrome found for 1673 at  99 after 11 steps.
Palindrome found for 1674 at  999 after 10 steps.
Palindrome found for 1675 at  909 after 4 steps.
Palindrome found for 1676 at  99 after 6 steps.
Palindrome found for 1677 at  999 after 7 steps.
Palindrome found for 1678 at  999 after 7 steps.
Palindrome found for 1679 at  99 after 7 steps.
Palindrome found for 1680 at  99 after 2 steps.
Palindrome found for 1681 at  99 after 2 steps.
Palindrome found for 1682 at  999 after 11 steps.
limit exceeded for 1683
Palindrome found for 1684 at  909 after 7 steps.
Palindrome found for 1685 at  99 after 10 steps.
Palindrome found for 1686 at  99 after 4 steps.
Palindrome found for 1687 at  999 after 9 steps.
Palindrome found for 1688 at  999 after 12 steps.
Palindrome found for 1689 at  909 after 3 steps.
Palindrome found for 1690 at  99 after 6 steps.
Palindrome found for 1691 at  99 after 6 steps.
Palindrome found for 1692 at  99 after 10 steps.
Palindrome found for 1693

Palindrome found for 1902 at  99 after 4 steps.
limit exceeded for 1903
Palindrome found for 1904 at  99 after 6 steps.
Palindrome found for 1905 at  909 after 6 steps.
Palindrome found for 1906 at  99 after 8 steps.
Palindrome found for 1907 at  99 after 4 steps.
Palindrome found for 1908 at  99 after 7 steps.
Palindrome found for 1909 at  99 after 12 steps.
Palindrome found for 1910 at  99 after 7 steps.
Palindrome found for 1911 at  99 after 5 steps.
Palindrome found for 1912 at  99 after 5 steps.
Palindrome found for 1913 at  999 after 6 steps.
Palindrome found for 1914 at  5445 after 2 steps.
Palindrome found for 1915 at  999 after 6 steps.
Palindrome found for 1916 at  999 after 5 steps.
Palindrome found for 1917 at  99 after 6 steps.
Palindrome found for 1918 at  99 after 7 steps.
Palindrome found for 1919 at  909 after 3 steps.
Palindrome found for 1920 at  99 after 7 steps.
Palindrome found for 1921 at  99 after 3 steps.
Palindrome found for 1922 at  99 after 3 steps.
Palindro

limit exceeded for 2123
Palindrome found for 2124 at  99 after 11 steps.
Palindrome found for 2125 at  999 after 10 steps.
Palindrome found for 2126 at  909 after 4 steps.
Palindrome found for 2127 at  99 after 6 steps.
Palindrome found for 2128 at  999 after 7 steps.
Palindrome found for 2129 at  999 after 7 steps.
Palindrome found for 2130 at  909 after 5 steps.
Palindrome found for 2131 at  99 after 2 steps.
Palindrome found for 2132 at  99 after 2 steps.
Palindrome found for 2133 at  999 after 11 steps.
limit exceeded for 2134
Palindrome found for 2135 at  909 after 7 steps.
Palindrome found for 2136 at  99 after 10 steps.
Palindrome found for 2137 at  99 after 4 steps.
Palindrome found for 2138 at  999 after 9 steps.
Palindrome found for 2139 at  999 after 12 steps.
Palindrome found for 2140 at  999 after 12 steps.
Palindrome found for 2141 at  99 after 6 steps.
Palindrome found for 2142 at  99 after 6 steps.
Palindrome found for 2143 at  99 after 10 steps.
Palindrome found for 21

limit exceeded for 2343
Palindrome found for 2344 at  99 after 11 steps.
Palindrome found for 2345 at  999 after 10 steps.
Palindrome found for 2346 at  909 after 4 steps.
Palindrome found for 2347 at  99 after 6 steps.
Palindrome found for 2348 at  999 after 7 steps.
Palindrome found for 2349 at  999 after 7 steps.
Palindrome found for 2350 at  909 after 5 steps.
Palindrome found for 2351 at  99 after 2 steps.
Palindrome found for 2352 at  99 after 2 steps.
Palindrome found for 2353 at  999 after 11 steps.
limit exceeded for 2354
Palindrome found for 2355 at  909 after 7 steps.
Palindrome found for 2356 at  99 after 10 steps.
Palindrome found for 2357 at  99 after 4 steps.
Palindrome found for 2358 at  999 after 9 steps.
Palindrome found for 2359 at  999 after 12 steps.
Palindrome found for 2360 at  999 after 12 steps.
Palindrome found for 2361 at  99 after 6 steps.
Palindrome found for 2362 at  99 after 6 steps.
Palindrome found for 2363 at  99 after 10 steps.
Palindrome found for 23

limit exceeded for 2541
Palindrome found for 2542 at  9 after 6 steps.
Palindrome found for 2543 at  909 after 1 steps.
Palindrome found for 2544 at  99 after 8 steps.
Palindrome found for 2545 at  99 after 9 steps.
Palindrome found for 2546 at  99 after 7 steps.
Palindrome found for 2547 at  99 after 5 steps.
Palindrome found for 2548 at  909 after 5 steps.
Palindrome found for 2549 at  999 after 8 steps.
Palindrome found for 2550 at  999 after 5 steps.
Palindrome found for 2551 at  999 after 1 steps.
Palindrome found for 2552 at  2552 after 0 steps.
Palindrome found for 2553 at  999 after 1 steps.
Palindrome found for 2554 at  999 after 5 steps.
Palindrome found for 2555 at  999 after 3 steps.
Palindrome found for 2556 at  999 after 4 steps.
Palindrome found for 2557 at  999 after 2 steps.
Palindrome found for 2558 at  999 after 2 steps.
Palindrome found for 2559 at  999 after 4 steps.
Palindrome found for 2560 at  99 after 8 steps.
Palindrome found for 2561 at  909 after 1 steps.
Pa

limit exceeded for 2750
Palindrome found for 2751 at  999 after 11 steps.
Palindrome found for 2752 at  99 after 2 steps.
Palindrome found for 2753 at  99 after 2 steps.
Palindrome found for 2754 at  909 after 5 steps.
Palindrome found for 2755 at  99 after 12 steps.
Palindrome found for 2756 at  99 after 7 steps.
Palindrome found for 2757 at  99 after 4 steps.
Palindrome found for 2758 at  99 after 8 steps.
Palindrome found for 2759 at  909 after 6 steps.
Palindrome found for 2760 at  99 after 11 steps.
limit exceeded for 2761
Palindrome found for 2762 at  9 after 6 steps.
Palindrome found for 2763 at  909 after 1 steps.
Palindrome found for 2764 at  99 after 8 steps.
Palindrome found for 2765 at  99 after 9 steps.
Palindrome found for 2766 at  99 after 7 steps.
Palindrome found for 2767 at  99 after 5 steps.
Palindrome found for 2768 at  909 after 5 steps.
Palindrome found for 2769 at  999 after 8 steps.
Palindrome found for 2770 at  999 after 5 steps.
Palindrome found for 2771 at  9

limit exceeded for 2959
Palindrome found for 2960 at  909 after 7 steps.
Palindrome found for 2961 at  99 after 10 steps.
Palindrome found for 2962 at  99 after 6 steps.
Palindrome found for 2963 at  99 after 6 steps.
Palindrome found for 2964 at  999 after 12 steps.
Palindrome found for 2965 at  909 after 3 steps.
Palindrome found for 2966 at  99 after 7 steps.
Palindrome found for 2967 at  99 after 6 steps.
Palindrome found for 2968 at  999 after 5 steps.
Palindrome found for 2969 at  999 after 6 steps.
limit exceeded for 2970
Palindrome found for 2971 at  999 after 11 steps.
Palindrome found for 2972 at  99 after 2 steps.
Palindrome found for 2973 at  99 after 2 steps.
Palindrome found for 2974 at  909 after 5 steps.
Palindrome found for 2975 at  99 after 12 steps.
Palindrome found for 2976 at  99 after 7 steps.
Palindrome found for 2977 at  99 after 4 steps.
Palindrome found for 2978 at  99 after 8 steps.
Palindrome found for 2979 at  909 after 6 steps.
Palindrome found for 2980 at

limit exceeded for 3135
Palindrome found for 3136 at  909 after 7 steps.
Palindrome found for 3137 at  99 after 10 steps.
Palindrome found for 3138 at  99 after 4 steps.
Palindrome found for 3139 at  999 after 9 steps.
Palindrome found for 3140 at  909 after 3 steps.
Palindrome found for 3141 at  999 after 12 steps.
Palindrome found for 3142 at  99 after 6 steps.
Palindrome found for 3143 at  99 after 6 steps.
Palindrome found for 3144 at  99 after 10 steps.
Palindrome found for 3145 at  909 after 7 steps.
limit exceeded for 3146
Palindrome found for 3147 at  999 after 11 steps.
Palindrome found for 3148 at  99 after 5 steps.
Palindrome found for 3149 at  999 after 8 steps.
Palindrome found for 3150 at  99 after 7 steps.
Palindrome found for 3151 at  999 after 7 steps.
Palindrome found for 3152 at  99 after 4 steps.
Palindrome found for 3153 at  99 after 4 steps.
Palindrome found for 3154 at  909 after 4 steps.
Palindrome found for 3155 at  999 after 10 steps.
Palindrome found for 3156

limit exceeded for 3311
Palindrome found for 3312 at  999 after 11 steps.
Palindrome found for 3313 at  99 after 2 steps.
Palindrome found for 3314 at  99 after 2 steps.
Palindrome found for 3315 at  909 after 5 steps.
Palindrome found for 3316 at  99 after 12 steps.
Palindrome found for 3317 at  99 after 7 steps.
Palindrome found for 3318 at  99 after 4 steps.
Palindrome found for 3319 at  99 after 8 steps.
Palindrome found for 3320 at  999 after 10 steps.
Palindrome found for 3321 at  99 after 11 steps.
limit exceeded for 3322
Palindrome found for 3323 at  9 after 6 steps.
Palindrome found for 3324 at  909 after 1 steps.
Palindrome found for 3325 at  99 after 8 steps.
Palindrome found for 3326 at  99 after 9 steps.
Palindrome found for 3327 at  99 after 7 steps.
Palindrome found for 3328 at  99 after 5 steps.
Palindrome found for 3329 at  909 after 5 steps.
Palindrome found for 3330 at  999 after 3 steps.
Palindrome found for 3331 at  999 after 5 steps.
Palindrome found for 3332 at  

limit exceeded for 3542
Palindrome found for 3543 at  9 after 6 steps.
Palindrome found for 3544 at  909 after 1 steps.
Palindrome found for 3545 at  99 after 8 steps.
Palindrome found for 3546 at  99 after 9 steps.
Palindrome found for 3547 at  99 after 7 steps.
Palindrome found for 3548 at  99 after 5 steps.
Palindrome found for 3549 at  909 after 5 steps.
Palindrome found for 3550 at  999 after 3 steps.
Palindrome found for 3551 at  999 after 5 steps.
Palindrome found for 3552 at  999 after 1 steps.
Palindrome found for 3553 at  3553 after 0 steps.
Palindrome found for 3554 at  999 after 1 steps.
Palindrome found for 3555 at  999 after 5 steps.
Palindrome found for 3556 at  999 after 3 steps.
Palindrome found for 3557 at  999 after 4 steps.
Palindrome found for 3558 at  999 after 2 steps.
Palindrome found for 3559 at  999 after 2 steps.
Palindrome found for 3560 at  99 after 9 steps.
Palindrome found for 3561 at  99 after 8 steps.
Palindrome found for 3562 at  909 after 1 steps.
Pal

limit exceeded for 3751
Palindrome found for 3752 at  999 after 11 steps.
Palindrome found for 3753 at  99 after 2 steps.
Palindrome found for 3754 at  99 after 2 steps.
Palindrome found for 3755 at  909 after 5 steps.
Palindrome found for 3756 at  99 after 12 steps.
Palindrome found for 3757 at  99 after 7 steps.
Palindrome found for 3758 at  99 after 4 steps.
Palindrome found for 3759 at  99 after 8 steps.
Palindrome found for 3760 at  999 after 10 steps.
Palindrome found for 3761 at  99 after 11 steps.
limit exceeded for 3762
Palindrome found for 3763 at  9 after 6 steps.
Palindrome found for 3764 at  909 after 1 steps.
Palindrome found for 3765 at  99 after 8 steps.
Palindrome found for 3766 at  99 after 9 steps.
Palindrome found for 3767 at  99 after 7 steps.
Palindrome found for 3768 at  99 after 5 steps.
Palindrome found for 3769 at  909 after 5 steps.
Palindrome found for 3770 at  999 after 3 steps.
Palindrome found for 3771 at  999 after 5 steps.
Palindrome found for 3772 at  

limit exceeded for 3927
Palindrome found for 3928 at  99 after 11 steps.
Palindrome found for 3929 at  99 after 7 steps.
Palindrome found for 3930 at  99 after 8 steps.
Palindrome found for 3931 at  99 after 9 steps.
Palindrome found for 3932 at  99 after 7 steps.
Palindrome found for 3933 at  99 after 3 steps.
Palindrome found for 3934 at  99 after 3 steps.
Palindrome found for 3935 at  999 after 8 steps.
Palindrome found for 3936 at  99 after 7 steps.
Palindrome found for 3937 at  999 after 11 steps.
limit exceeded for 3938
Palindrome found for 3939 at  909 after 2 steps.
Palindrome found for 3940 at  999 after 5 steps.
Palindrome found for 3941 at  999 after 3 steps.
Palindrome found for 3942 at  999 after 4 steps.
Palindrome found for 3943 at  99 after 5 steps.
Palindrome found for 3944 at  99 after 5 steps.
Palindrome found for 3945 at  999 after 9 steps.
Palindrome found for 3946 at  99 after 6 steps.
Palindrome found for 3947 at  99 after 10 steps.
Palindrome found for 3948 at  

limit exceeded for 4158
Palindrome found for 4159 at  99 after 3 steps.
Palindrome found for 4160 at  99 after 10 steps.
Palindrome found for 4161 at  99 after 6 steps.
Palindrome found for 4162 at  999 after 9 steps.
Palindrome found for 4163 at  99 after 5 steps.
Palindrome found for 4164 at  99 after 5 steps.
Palindrome found for 4165 at  999 after 4 steps.
Palindrome found for 4166 at  999 after 3 steps.
Palindrome found for 4167 at  999 after 5 steps.
Palindrome found for 4168 at  999 after 6 steps.
Palindrome found for 4169 at  5445 after 1 steps.
Palindrome found for 4170 at  999 after 11 steps.
Palindrome found for 4171 at  99 after 7 steps.
Palindrome found for 4172 at  999 after 8 steps.
Palindrome found for 4173 at  99 after 3 steps.
Palindrome found for 4174 at  99 after 3 steps.
Palindrome found for 4175 at  99 after 7 steps.
Palindrome found for 4176 at  99 after 9 steps.
Palindrome found for 4177 at  99 after 8 steps.
Palindrome found for 4178 at  909 after 6 steps.
Pali

Palindrome found for 4379 at  99 after 3 steps.
Palindrome found for 4380 at  99 after 10 steps.
Palindrome found for 4381 at  99 after 6 steps.
Palindrome found for 4382 at  999 after 9 steps.
Palindrome found for 4383 at  99 after 5 steps.
Palindrome found for 4384 at  99 after 5 steps.
Palindrome found for 4385 at  999 after 4 steps.
Palindrome found for 4386 at  999 after 3 steps.
Palindrome found for 4387 at  999 after 5 steps.
Palindrome found for 4388 at  999 after 6 steps.
Palindrome found for 4389 at  5445 after 1 steps.
Palindrome found for 4390 at  999 after 11 steps.
Palindrome found for 4391 at  99 after 7 steps.
Palindrome found for 4392 at  999 after 8 steps.
Palindrome found for 4393 at  99 after 3 steps.
Palindrome found for 4394 at  99 after 3 steps.
Palindrome found for 4395 at  99 after 7 steps.
Palindrome found for 4396 at  99 after 9 steps.
Palindrome found for 4397 at  99 after 8 steps.
Palindrome found for 4398 at  909 after 6 steps.
Palindrome found for 4399 at

limit exceeded for 4587
Palindrome found for 4588 at  999 after 11 steps.
Palindrome found for 4589 at  99 after 5 steps.
Palindrome found for 4590 at  909 after 4 steps.
Palindrome found for 4591 at  99 after 7 steps.
Palindrome found for 4592 at  999 after 7 steps.
Palindrome found for 4593 at  99 after 4 steps.
Palindrome found for 4594 at  99 after 4 steps.
Palindrome found for 4595 at  909 after 4 steps.
Palindrome found for 4596 at  999 after 10 steps.
Palindrome found for 4597 at  99 after 11 steps.
limit exceeded for 4598
Palindrome found for 4599 at  99 after 3 steps.
Palindrome found for 4600 at  909 after 6 steps.
Palindrome found for 4601 at  99 after 8 steps.
Palindrome found for 4602 at  99 after 9 steps.
Palindrome found for 4603 at  99 after 7 steps.
Palindrome found for 4604 at  99 after 3 steps.
Palindrome found for 4605 at  99 after 3 steps.
Palindrome found for 4606 at  999 after 8 steps.
Palindrome found for 4607 at  99 after 7 steps.
Palindrome found for 4608 at  

limit exceeded for 4785
Palindrome found for 4786 at  99 after 11 steps.
Palindrome found for 4787 at  999 after 10 steps.
Palindrome found for 4788 at  909 after 4 steps.
Palindrome found for 4789 at  99 after 6 steps.
Palindrome found for 4790 at  99 after 7 steps.
Palindrome found for 4791 at  99 after 12 steps.
Palindrome found for 4792 at  909 after 5 steps.
Palindrome found for 4793 at  99 after 2 steps.
Palindrome found for 4794 at  99 after 2 steps.
Palindrome found for 4795 at  999 after 11 steps.
limit exceeded for 4796
Palindrome found for 4797 at  909 after 7 steps.
Palindrome found for 4798 at  99 after 10 steps.
Palindrome found for 4799 at  99 after 4 steps.
Palindrome found for 4800 at  999 after 9 steps.
Palindrome found for 4801 at  999 after 12 steps.
Palindrome found for 4802 at  909 after 3 steps.
Palindrome found for 4803 at  99 after 7 steps.
Palindrome found for 4804 at  99 after 5 steps.
Palindrome found for 4805 at  99 after 5 steps.
Palindrome found for 4806 

limit exceeded for 5027
Palindrome found for 5028 at  909 after 7 steps.
Palindrome found for 5029 at  99 after 10 steps.
Palindrome found for 5030 at  99 after 6 steps.
Palindrome found for 5031 at  99 after 7 steps.
Palindrome found for 5032 at  909 after 3 steps.
Palindrome found for 5033 at  999 after 12 steps.
Palindrome found for 5034 at  99 after 6 steps.
Palindrome found for 5035 at  99 after 6 steps.
Palindrome found for 5036 at  99 after 10 steps.
Palindrome found for 5037 at  909 after 7 steps.
limit exceeded for 5038
Palindrome found for 5039 at  999 after 11 steps.
Palindrome found for 5040 at  99 after 7 steps.
Palindrome found for 5041 at  909 after 4 steps.
Palindrome found for 5042 at  99 after 7 steps.
Palindrome found for 5043 at  999 after 7 steps.
Palindrome found for 5044 at  99 after 4 steps.
Palindrome found for 5045 at  99 after 4 steps.
Palindrome found for 5046 at  909 after 4 steps.
Palindrome found for 5047 at  999 after 10 steps.
Palindrome found for 5048 

limit exceeded for 5247
Palindrome found for 5248 at  909 after 7 steps.
Palindrome found for 5249 at  99 after 10 steps.
Palindrome found for 5250 at  99 after 6 steps.
Palindrome found for 5251 at  99 after 7 steps.
Palindrome found for 5252 at  909 after 3 steps.
Palindrome found for 5253 at  999 after 12 steps.
Palindrome found for 5254 at  99 after 6 steps.
Palindrome found for 5255 at  99 after 6 steps.
Palindrome found for 5256 at  99 after 10 steps.
Palindrome found for 5257 at  909 after 7 steps.
limit exceeded for 5258
Palindrome found for 5259 at  999 after 11 steps.
Palindrome found for 5260 at  99 after 7 steps.
Palindrome found for 5261 at  909 after 4 steps.
Palindrome found for 5262 at  99 after 7 steps.
Palindrome found for 5263 at  999 after 7 steps.
Palindrome found for 5264 at  99 after 4 steps.
Palindrome found for 5265 at  99 after 4 steps.
Palindrome found for 5266 at  909 after 4 steps.
Palindrome found for 5267 at  999 after 10 steps.
Palindrome found for 5268 

limit exceeded for 5467
Palindrome found for 5468 at  909 after 7 steps.
Palindrome found for 5469 at  99 after 10 steps.
Palindrome found for 5470 at  99 after 6 steps.
Palindrome found for 5471 at  99 after 7 steps.
Palindrome found for 5472 at  909 after 3 steps.
Palindrome found for 5473 at  999 after 12 steps.
Palindrome found for 5474 at  99 after 6 steps.
Palindrome found for 5475 at  99 after 6 steps.
Palindrome found for 5476 at  99 after 10 steps.
Palindrome found for 5477 at  909 after 7 steps.
limit exceeded for 5478
Palindrome found for 5479 at  999 after 11 steps.
Palindrome found for 5480 at  99 after 7 steps.
Palindrome found for 5481 at  909 after 4 steps.
Palindrome found for 5482 at  99 after 7 steps.
Palindrome found for 5483 at  999 after 7 steps.
Palindrome found for 5484 at  99 after 4 steps.
Palindrome found for 5485 at  99 after 4 steps.
Palindrome found for 5486 at  909 after 4 steps.
Palindrome found for 5487 at  999 after 10 steps.
Palindrome found for 5488 

limit exceeded for 5698
Palindrome found for 5699 at  999 after 11 steps.
Palindrome found for 5700 at  99 after 5 steps.
Palindrome found for 5701 at  999 after 8 steps.
Palindrome found for 5702 at  909 after 5 steps.
Palindrome found for 5703 at  99 after 12 steps.
Palindrome found for 5704 at  99 after 7 steps.
Palindrome found for 5705 at  99 after 3 steps.
Palindrome found for 5706 at  99 after 3 steps.
Palindrome found for 5707 at  909 after 6 steps.
Palindrome found for 5708 at  99 after 6 steps.
limit exceeded for 5709
Palindrome found for 5710 at  99 after 3 steps.
Palindrome found for 5711 at  909 after 6 steps.
Palindrome found for 5712 at  99 after 8 steps.
Palindrome found for 5713 at  99 after 9 steps.
Palindrome found for 5714 at  99 after 7 steps.
Palindrome found for 5715 at  99 after 3 steps.
Palindrome found for 5716 at  99 after 3 steps.
Palindrome found for 5717 at  999 after 8 steps.
Palindrome found for 5718 at  99 after 7 steps.
Palindrome found for 5719 at  99

limit exceeded for 5874
Palindrome found for 5875 at  9 after 6 steps.
Palindrome found for 5876 at  909 after 1 steps.
Palindrome found for 5877 at  99 after 8 steps.
Palindrome found for 5878 at  99 after 9 steps.
Palindrome found for 5879 at  99 after 7 steps.
Palindrome found for 5880 at  999 after 2 steps.
Palindrome found for 5881 at  999 after 4 steps.
Palindrome found for 5882 at  999 after 3 steps.
Palindrome found for 5883 at  999 after 5 steps.
Palindrome found for 5884 at  999 after 1 steps.
Palindrome found for 5885 at  5885 after 0 steps.
Palindrome found for 5886 at  999 after 1 steps.
Palindrome found for 5887 at  999 after 5 steps.
Palindrome found for 5888 at  999 after 3 steps.
Palindrome found for 5889 at  999 after 4 steps.
Palindrome found for 5890 at  99 after 5 steps.
Palindrome found for 5891 at  99 after 7 steps.
Palindrome found for 5892 at  99 after 9 steps.
Palindrome found for 5893 at  99 after 8 steps.
Palindrome found for 5894 at  909 after 1 steps.
Pali

limit exceeded for 6050
Palindrome found for 6051 at  909 after 2 steps.
Palindrome found for 6052 at  99 after 10 steps.
Palindrome found for 6053 at  99 after 6 steps.
Palindrome found for 6054 at  999 after 9 steps.
Palindrome found for 6055 at  99 after 5 steps.
Palindrome found for 6056 at  99 after 5 steps.
Palindrome found for 6057 at  999 after 4 steps.
Palindrome found for 6058 at  999 after 3 steps.
Palindrome found for 6059 at  999 after 5 steps.
Palindrome found for 6060 at  909 after 2 steps.
limit exceeded for 6061
Palindrome found for 6062 at  999 after 11 steps.
Palindrome found for 6063 at  99 after 7 steps.
Palindrome found for 6064 at  999 after 8 steps.
Palindrome found for 6065 at  99 after 3 steps.
Palindrome found for 6066 at  99 after 3 steps.
Palindrome found for 6067 at  99 after 7 steps.
Palindrome found for 6068 at  99 after 9 steps.
Palindrome found for 6069 at  99 after 8 steps.
Palindrome found for 6070 at  99 after 7 steps.
Palindrome found for 6071 at  

limit exceeded for 6281
Palindrome found for 6282 at  999 after 11 steps.
Palindrome found for 6283 at  99 after 7 steps.
Palindrome found for 6284 at  999 after 8 steps.
Palindrome found for 6285 at  99 after 3 steps.
Palindrome found for 6286 at  99 after 3 steps.
Palindrome found for 6287 at  99 after 7 steps.
Palindrome found for 6288 at  99 after 9 steps.
Palindrome found for 6289 at  99 after 8 steps.
Palindrome found for 6290 at  99 after 7 steps.
Palindrome found for 6291 at  99 after 11 steps.
limit exceeded for 6292
Palindrome found for 6293 at  99 after 6 steps.
Palindrome found for 6294 at  909 after 6 steps.
Palindrome found for 6295 at  99 after 3 steps.
Palindrome found for 6296 at  99 after 3 steps.
Palindrome found for 6297 at  99 after 7 steps.
Palindrome found for 6298 at  99 after 12 steps.
Palindrome found for 6299 at  909 after 5 steps.
Palindrome found for 6300 at  999 after 8 steps.
Palindrome found for 6301 at  99 after 5 steps.
Palindrome found for 6302 at  99

Palindrome found for 6473 at  909 after 3 steps.
Palindrome found for 6474 at  999 after 12 steps.
Palindrome found for 6475 at  99 after 6 steps.
Palindrome found for 6476 at  99 after 6 steps.
Palindrome found for 6477 at  99 after 10 steps.
Palindrome found for 6478 at  909 after 7 steps.
limit exceeded for 6479
Palindrome found for 6480 at  99 after 11 steps.
Palindrome found for 6481 at  99 after 7 steps.
Palindrome found for 6482 at  909 after 4 steps.
Palindrome found for 6483 at  99 after 7 steps.
Palindrome found for 6484 at  999 after 7 steps.
Palindrome found for 6485 at  99 after 4 steps.
Palindrome found for 6486 at  99 after 4 steps.
Palindrome found for 6487 at  909 after 4 steps.
Palindrome found for 6488 at  999 after 10 steps.
Palindrome found for 6489 at  99 after 11 steps.
limit exceeded for 6490
Palindrome found for 6491 at  909 after 2 steps.
Palindrome found for 6492 at  99 after 10 steps.
Palindrome found for 6493 at  99 after 6 steps.
Palindrome found for 6494 

limit exceeded for 6710
Palindrome found for 6711 at  99 after 3 steps.
Palindrome found for 6712 at  909 after 6 steps.
Palindrome found for 6713 at  99 after 8 steps.
Palindrome found for 6714 at  99 after 9 steps.
Palindrome found for 6715 at  99 after 7 steps.
Palindrome found for 6716 at  99 after 3 steps.
Palindrome found for 6717 at  99 after 3 steps.
Palindrome found for 6718 at  999 after 8 steps.
Palindrome found for 6719 at  99 after 7 steps.
Palindrome found for 6720 at  999 after 6 steps.
Palindrome found for 6721 at  5445 after 1 steps.
Palindrome found for 6722 at  999 after 6 steps.
Palindrome found for 6723 at  999 after 5 steps.
Palindrome found for 6724 at  999 after 3 steps.
Palindrome found for 6725 at  999 after 4 steps.
Palindrome found for 6726 at  99 after 5 steps.
Palindrome found for 6727 at  99 after 5 steps.
Palindrome found for 6728 at  999 after 9 steps.
Palindrome found for 6729 at  99 after 6 steps.
Palindrome found for 6730 at  909 after 6 steps.
Palin

limit exceeded for 6963
Palindrome found for 6964 at  909 after 7 steps.
Palindrome found for 6965 at  99 after 10 steps.
Palindrome found for 6966 at  99 after 6 steps.
Palindrome found for 6967 at  99 after 6 steps.
Palindrome found for 6968 at  999 after 12 steps.
Palindrome found for 6969 at  909 after 3 steps.
Palindrome found for 6970 at  999 after 9 steps.
Palindrome found for 6971 at  99 after 4 steps.
Palindrome found for 6972 at  99 after 10 steps.
Palindrome found for 6973 at  909 after 7 steps.
limit exceeded for 6974
Palindrome found for 6975 at  999 after 11 steps.
Palindrome found for 6976 at  99 after 2 steps.
Palindrome found for 6977 at  99 after 2 steps.
Palindrome found for 6978 at  909 after 5 steps.
Palindrome found for 6979 at  99 after 12 steps.
Palindrome found for 6980 at  999 after 7 steps.
Palindrome found for 6981 at  99 after 6 steps.
Palindrome found for 6982 at  909 after 4 steps.
Palindrome found for 6983 at  999 after 10 steps.
Palindrome found for 698

Palindrome found for 7206 at  999 after 11 steps.
Palindrome found for 7207 at  99 after 2 steps.
Palindrome found for 7208 at  99 after 2 steps.
Palindrome found for 7209 at  909 after 5 steps.
Palindrome found for 7210 at  999 after 7 steps.
Palindrome found for 7211 at  999 after 7 steps.
Palindrome found for 7212 at  99 after 6 steps.
Palindrome found for 7213 at  909 after 4 steps.
Palindrome found for 7214 at  999 after 10 steps.
Palindrome found for 7215 at  99 after 11 steps.
limit exceeded for 7216
Palindrome found for 7217 at  9 after 6 steps.
Palindrome found for 7218 at  909 after 1 steps.
Palindrome found for 7219 at  99 after 8 steps.
Palindrome found for 7220 at  999 after 4 steps.
Palindrome found for 7221 at  999 after 2 steps.
Palindrome found for 7222 at  999 after 2 steps.
Palindrome found for 7223 at  999 after 4 steps.
Palindrome found for 7224 at  999 after 3 steps.
Palindrome found for 7225 at  999 after 5 steps.
Palindrome found for 7226 at  999 after 1 steps.


limit exceeded for 7403
Palindrome found for 7404 at  99 after 11 steps.
Palindrome found for 7405 at  999 after 10 steps.
Palindrome found for 7406 at  909 after 4 steps.
Palindrome found for 7407 at  99 after 4 steps.
Palindrome found for 7408 at  99 after 4 steps.
Palindrome found for 7409 at  999 after 7 steps.
Palindrome found for 7410 at  909 after 5 steps.
Palindrome found for 7411 at  999 after 8 steps.
Palindrome found for 7412 at  99 after 5 steps.
Palindrome found for 7413 at  999 after 11 steps.
limit exceeded for 7414
Palindrome found for 7415 at  909 after 7 steps.
Palindrome found for 7416 at  99 after 10 steps.
Palindrome found for 7417 at  99 after 6 steps.
Palindrome found for 7418 at  99 after 6 steps.
Palindrome found for 7419 at  999 after 12 steps.
Palindrome found for 7420 at  999 after 12 steps.
Palindrome found for 7421 at  999 after 9 steps.
Palindrome found for 7422 at  99 after 4 steps.
Palindrome found for 7423 at  99 after 10 steps.
Palindrome found for 74

Palindrome found for 7609 at  999 after 8 steps.
Palindrome found for 7610 at  999 after 5 steps.
Palindrome found for 7611 at  999 after 6 steps.
Palindrome found for 7612 at  5445 after 1 steps.
Palindrome found for 7613 at  999 after 6 steps.
Palindrome found for 7614 at  999 after 5 steps.
Palindrome found for 7615 at  999 after 3 steps.
Palindrome found for 7616 at  999 after 4 steps.
Palindrome found for 7617 at  99 after 5 steps.
Palindrome found for 7618 at  99 after 5 steps.
Palindrome found for 7619 at  999 after 9 steps.
Palindrome found for 7620 at  99 after 8 steps.
Palindrome found for 7621 at  909 after 6 steps.
Palindrome found for 7622 at  99 after 3 steps.
limit exceeded for 7623
Palindrome found for 7624 at  99 after 11 steps.
Palindrome found for 7625 at  999 after 10 steps.
Palindrome found for 7626 at  909 after 4 steps.
Palindrome found for 7627 at  99 after 4 steps.
Palindrome found for 7628 at  99 after 4 steps.
Palindrome found for 7629 at  999 after 7 steps.


limit exceeded for 7788
Palindrome found for 7789 at  99 after 11 steps.
Palindrome found for 7790 at  909 after 6 steps.
Palindrome found for 7791 at  99 after 8 steps.
Palindrome found for 7792 at  99 after 4 steps.
Palindrome found for 7793 at  99 after 7 steps.
Palindrome found for 7794 at  99 after 12 steps.
Palindrome found for 7795 at  909 after 5 steps.
Palindrome found for 7796 at  99 after 2 steps.
Palindrome found for 7797 at  99 after 2 steps.
Palindrome found for 7798 at  999 after 11 steps.
limit exceeded for 7799
Palindrome found for 7800 at  909 after 7 steps.
Palindrome found for 7801 at  99 after 10 steps.
Palindrome found for 7802 at  99 after 4 steps.
Palindrome found for 7803 at  999 after 9 steps.
Palindrome found for 7804 at  999 after 12 steps.
Palindrome found for 7805 at  909 after 3 steps.
Palindrome found for 7806 at  99 after 7 steps.
Palindrome found for 7807 at  99 after 5 steps.
Palindrome found for 7808 at  99 after 5 steps.
Palindrome found for 7809 at

limit exceeded for 8041
Palindrome found for 8042 at  99 after 11 steps.
Palindrome found for 8043 at  99 after 7 steps.
Palindrome found for 8044 at  909 after 4 steps.
Palindrome found for 8045 at  99 after 7 steps.
Palindrome found for 8046 at  999 after 7 steps.
Palindrome found for 8047 at  99 after 4 steps.
Palindrome found for 8048 at  99 after 4 steps.
Palindrome found for 8049 at  909 after 4 steps.
Palindrome found for 8050 at  99 after 7 steps.
Palindrome found for 8051 at  999 after 11 steps.
limit exceeded for 8052
Palindrome found for 8053 at  909 after 2 steps.
Palindrome found for 8054 at  99 after 10 steps.
Palindrome found for 8055 at  99 after 6 steps.
Palindrome found for 8056 at  999 after 9 steps.
Palindrome found for 8057 at  99 after 5 steps.
Palindrome found for 8058 at  99 after 5 steps.
Palindrome found for 8059 at  999 after 4 steps.
Palindrome found for 8060 at  99 after 6 steps.
Palindrome found for 8061 at  99 after 10 steps.
Palindrome found for 8062 at 

limit exceeded for 8294
Palindrome found for 8295 at  99 after 6 steps.
Palindrome found for 8296 at  909 after 6 steps.
Palindrome found for 8297 at  99 after 3 steps.
Palindrome found for 8298 at  99 after 3 steps.
Palindrome found for 8299 at  99 after 7 steps.
Palindrome found for 8300 at  99 after 12 steps.
Palindrome found for 8301 at  909 after 5 steps.
Palindrome found for 8302 at  999 after 8 steps.
Palindrome found for 8303 at  99 after 5 steps.
Palindrome found for 8304 at  999 after 11 steps.
limit exceeded for 8305
Palindrome found for 8306 at  909 after 7 steps.
Palindrome found for 8307 at  99 after 10 steps.
Palindrome found for 8308 at  99 after 6 steps.
Palindrome found for 8309 at  99 after 6 steps.
Palindrome found for 8310 at  909 after 3 steps.
Palindrome found for 8311 at  999 after 12 steps.
Palindrome found for 8312 at  999 after 9 steps.
Palindrome found for 8313 at  99 after 4 steps.
Palindrome found for 8314 at  99 after 10 steps.
Palindrome found for 8315 a

limit exceeded for 8525
Palindrome found for 8526 at  909 after 7 steps.
Palindrome found for 8527 at  99 after 10 steps.
Palindrome found for 8528 at  99 after 6 steps.
Palindrome found for 8529 at  99 after 6 steps.
Palindrome found for 8530 at  909 after 3 steps.
Palindrome found for 8531 at  999 after 12 steps.
Palindrome found for 8532 at  999 after 9 steps.
Palindrome found for 8533 at  99 after 4 steps.
Palindrome found for 8534 at  99 after 10 steps.
Palindrome found for 8535 at  909 after 7 steps.
limit exceeded for 8536
Palindrome found for 8537 at  999 after 11 steps.
Palindrome found for 8538 at  99 after 2 steps.
Palindrome found for 8539 at  99 after 2 steps.
Palindrome found for 8540 at  99 after 7 steps.
Palindrome found for 8541 at  999 after 7 steps.
Palindrome found for 8542 at  999 after 7 steps.
Palindrome found for 8543 at  99 after 6 steps.
Palindrome found for 8544 at  909 after 4 steps.
Palindrome found for 8545 at  999 after 10 steps.
Palindrome found for 8546

limit exceeded for 8767
Palindrome found for 8768 at  9 after 6 steps.
Palindrome found for 8769 at  909 after 1 steps.
Palindrome found for 8770 at  999 after 3 steps.
Palindrome found for 8771 at  999 after 4 steps.
Palindrome found for 8772 at  999 after 2 steps.
Palindrome found for 8773 at  999 after 2 steps.
Palindrome found for 8774 at  999 after 4 steps.
Palindrome found for 8775 at  999 after 3 steps.
Palindrome found for 8776 at  999 after 5 steps.
Palindrome found for 8777 at  999 after 1 steps.
Palindrome found for 8778 at  8778 after 0 steps.
Palindrome found for 8779 at  999 after 1 steps.
Palindrome found for 8780 at  99 after 7 steps.
Palindrome found for 8781 at  999 after 8 steps.
Palindrome found for 8782 at  909 after 5 steps.
Palindrome found for 8783 at  99 after 5 steps.
Palindrome found for 8784 at  99 after 7 steps.
Palindrome found for 8785 at  99 after 9 steps.
Palindrome found for 8786 at  99 after 8 steps.
Palindrome found for 8787 at  909 after 1 steps.
Pa

limit exceeded for 8987
Palindrome found for 8988 at  9 after 6 steps.
Palindrome found for 8989 at  909 after 1 steps.
Palindrome found for 8990 at  999 after 3 steps.
Palindrome found for 8991 at  999 after 4 steps.
Palindrome found for 8992 at  999 after 2 steps.
Palindrome found for 8993 at  999 after 2 steps.
Palindrome found for 8994 at  999 after 4 steps.
Palindrome found for 8995 at  999 after 3 steps.
Palindrome found for 8996 at  999 after 5 steps.
Palindrome found for 8997 at  999 after 1 steps.
Palindrome found for 8998 at  8998 after 0 steps.
Palindrome found for 8999 at  999 after 1 steps.
Palindrome found for 9000 at  999 after 5 steps.
Palindrome found for 9001 at  999 after 3 steps.
Palindrome found for 9002 at  999 after 4 steps.
Palindrome found for 9003 at  999 after 2 steps.
Palindrome found for 9004 at  999 after 2 steps.
Palindrome found for 9005 at  999 after 4 steps.
Palindrome found for 9006 at  999 after 3 steps.
Palindrome found for 9007 at  999 after 5 step

limit exceeded for 9174
Palindrome found for 9175 at  999 after 11 steps.
Palindrome found for 9176 at  99 after 7 steps.
Palindrome found for 9177 at  999 after 8 steps.
Palindrome found for 9178 at  99 after 3 steps.
Palindrome found for 9179 at  99 after 3 steps.
Palindrome found for 9180 at  999 after 7 steps.
Palindrome found for 9181 at  99 after 7 steps.
Palindrome found for 9182 at  909 after 4 steps.
Palindrome found for 9183 at  99 after 7 steps.
Palindrome found for 9184 at  99 after 11 steps.
limit exceeded for 9185
Palindrome found for 9186 at  99 after 6 steps.
Palindrome found for 9187 at  909 after 6 steps.
Palindrome found for 9188 at  99 after 3 steps.
Palindrome found for 9189 at  99 after 3 steps.
Palindrome found for 9190 at  999 after 12 steps.
Palindrome found for 9191 at  909 after 3 steps.
Palindrome found for 9192 at  99 after 7 steps.
Palindrome found for 9193 at  99 after 6 steps.
Palindrome found for 9194 at  999 after 5 steps.
Palindrome found for 9195 at 

limit exceeded for 9405
Palindrome found for 9406 at  99 after 11 steps.
Palindrome found for 9407 at  999 after 10 steps.
Palindrome found for 9408 at  909 after 4 steps.
Palindrome found for 9409 at  99 after 4 steps.
Palindrome found for 9410 at  99 after 7 steps.
Palindrome found for 9411 at  99 after 12 steps.
Palindrome found for 9412 at  909 after 5 steps.
Palindrome found for 9413 at  999 after 8 steps.
Palindrome found for 9414 at  99 after 5 steps.
Palindrome found for 9415 at  999 after 11 steps.
limit exceeded for 9416
Palindrome found for 9417 at  909 after 7 steps.
Palindrome found for 9418 at  99 after 10 steps.
Palindrome found for 9419 at  99 after 6 steps.
Palindrome found for 9420 at  99 after 7 steps.
Palindrome found for 9421 at  909 after 3 steps.
Palindrome found for 9422 at  999 after 12 steps.
Palindrome found for 9423 at  999 after 9 steps.
Palindrome found for 9424 at  99 after 4 steps.
Palindrome found for 9425 at  99 after 10 steps.
Palindrome found for 942

limit exceeded for 9592
Palindrome found for 9593 at  99 after 11 steps.
Palindrome found for 9594 at  99 after 7 steps.
Palindrome found for 9595 at  909 after 4 steps.
Palindrome found for 9596 at  99 after 7 steps.
Palindrome found for 9597 at  999 after 7 steps.
Palindrome found for 9598 at  99 after 4 steps.
Palindrome found for 9599 at  99 after 4 steps.
Palindrome found for 9600 at  909 after 4 steps.
Palindrome found for 9601 at  999 after 10 steps.
Palindrome found for 9602 at  99 after 11 steps.
limit exceeded for 9603
Palindrome found for 9604 at  99 after 3 steps.
Palindrome found for 9605 at  909 after 6 steps.
Palindrome found for 9606 at  99 after 8 steps.
Palindrome found for 9607 at  99 after 9 steps.
Palindrome found for 9608 at  99 after 7 steps.
Palindrome found for 9609 at  99 after 3 steps.
Palindrome found for 9610 at  999 after 4 steps.
Palindrome found for 9611 at  999 after 3 steps.
Palindrome found for 9612 at  999 after 5 steps.
Palindrome found for 9613 at 

limit exceeded for 9801
Palindrome found for 9802 at  909 after 7 steps.
Palindrome found for 9803 at  99 after 10 steps.
Palindrome found for 9804 at  99 after 4 steps.
Palindrome found for 9805 at  999 after 9 steps.
Palindrome found for 9806 at  999 after 12 steps.
Palindrome found for 9807 at  909 after 3 steps.
Palindrome found for 9808 at  99 after 7 steps.
Palindrome found for 9809 at  99 after 5 steps.
Palindrome found for 9810 at  99 after 10 steps.
Palindrome found for 9811 at  909 after 7 steps.
limit exceeded for 9812
Palindrome found for 9813 at  999 after 11 steps.
Palindrome found for 9814 at  99 after 5 steps.
Palindrome found for 9815 at  999 after 8 steps.
Palindrome found for 9816 at  909 after 5 steps.
Palindrome found for 9817 at  99 after 12 steps.
Palindrome found for 9818 at  99 after 7 steps.
Palindrome found for 9819 at  99 after 3 steps.
Palindrome found for 9820 at  909 after 4 steps.
Palindrome found for 9821 at  999 after 10 steps.
Palindrome found for 982

In [116]:
print(revlychrel_list)

[1012, 1023, 1034, 1045, 1067, 1078, 1089, 1100, 1122, 1133, 1144, 1155, 1177, 1188, 1199, 1210, 1232, 1243, 1254, 1265, 1287, 1298, 1320, 1342, 1353, 1364, 1375, 1397, 1408, 1430, 1452, 1463, 1474, 1485, 1507, 1518, 1540, 1562, 1573, 1584, 1595, 1606, 1617, 1628, 1650, 1672, 1683, 1694, 1705, 1716, 1727, 1738, 1760, 1782, 1793, 1815, 1826, 1837, 1848, 1870, 1892, 1903, 1925, 1936, 1947, 1958, 1980, 2013, 2024, 2035, 2046, 2068, 2079, 2090, 2101, 2123, 2134, 2145, 2156, 2178, 2189, 2200, 2211, 2233, 2244, 2255, 2266, 2288, 2299, 2310, 2321, 2343, 2354, 2365, 2376, 2398, 2409, 2420, 2431, 2453, 2464, 2475, 2486, 2508, 2519, 2530, 2541, 2563, 2574, 2585, 2596, 2607, 2618, 2629, 2640, 2651, 2673, 2684, 2695, 2706, 2717, 2728, 2739, 2750, 2761, 2783, 2794, 2816, 2827, 2838, 2849, 2860, 2871, 2893, 2904, 2926, 2937, 2948, 2959, 2970, 2981, 3014, 3025, 3036, 3047, 3069, 3091, 3102, 3124, 3135, 3146, 3157, 3179, 3201, 3212, 3234, 3245, 3256, 3267, 3289, 3300, 3311, 3322, 3344, 3355, 3366, 337

This is the list of Reverse Lychrels (numbers who do not form a palindrome when applied to the reverse process) .


In [121]:
forward_lychrel_data = pd.read_csv('datasets/LychrelTo100000Integers.txt',  header=None)
forward_lychrel_data = forward_lychrel_data.values.tolist()[0][1:]

unique_revlychrel = []
both = []

for rev_lychrel in revlychrel_list:
    valid = True
    for lychrel in forward_lychrel_data:
        if lychrel == rev_lychrel:
            valid = False
    if valid:
        unique_revlychrel.append(rev_lychrel)
    else:
        both.append(rev_lychrel)

In [120]:
print(unique_revlychrel)

[1012, 1023, 1034, 1045, 1067, 1078, 1089, 1100, 1122, 1133, 1144, 1155, 1177, 1188, 1199, 1210, 1232, 1243, 1254, 1265, 1287, 1298, 1320, 1342, 1353, 1364, 1375, 1397, 1408, 1430, 1452, 1463, 1474, 1485, 1507, 1518, 1540, 1562, 1573, 1584, 1595, 1606, 1617, 1628, 1650, 1672, 1683, 1694, 1705, 1716, 1727, 1738, 1760, 1782, 1793, 1815, 1826, 1837, 1848, 1870, 1892, 1903, 1925, 1936, 1958, 1980, 2013, 2024, 2035, 2046, 2068, 2079, 2090, 2101, 2123, 2134, 2145, 2156, 2178, 2189, 2200, 2211, 2233, 2244, 2255, 2266, 2288, 2299, 2310, 2321, 2343, 2354, 2365, 2376, 2398, 2409, 2420, 2431, 2453, 2464, 2475, 2486, 2508, 2519, 2530, 2541, 2563, 2574, 2585, 2596, 2607, 2618, 2629, 2640, 2651, 2673, 2684, 2695, 2706, 2717, 2728, 2739, 2750, 2761, 2783, 2794, 2816, 2827, 2838, 2849, 2860, 2871, 2893, 2904, 2926, 2937, 2948, 2959, 2970, 2981, 3014, 3025, 3036, 3047, 3069, 3091, 3102, 3124, 3135, 3146, 3157, 3179, 3201, 3212, 3234, 3245, 3256, 3267, 3289, 3300, 3311, 3322, 3344, 3355, 3366, 3377, 339

This is the list of Reverse Lychrels who are not Lychrels (will generate a palindrome regularly but not in reverse).

In [122]:
print(both)

[1947, 5258, 6347, 6490, 7436, 7491, 7689, 8349, 8525, 8899, 9438, 9867, 9988]


This is the list of numbers that act as Lychrels for both the regular and reverse process will not generate a palindrome in either case.

It is perhaps interesting to consider that this last list of numbers constitue what I am terming the 'true lychrels'. If we consider the reverse process to be the 'negative steps' of the lychrel process (i.e. 196 -691 is the negative 1st step in the process, which can perhaps even be refuted by pointing out that while 196 + 691 = 887 the opposite is not true as 887 - 788 ≠ 196), then 196 does indeed form a palindrome by the Lychrel process. However the numbers above never form a palindrome whether the process is applied to them 'regularly' or in reverse, thus they are 'true' Lychrel numbers as defined by not forming a palindrome by the Lychrel process.